CMP1903_A1_2324/Testing.cs

```diff
@@ -15,6 +15,19 @@ internal class Testing
          * Use debug.assert() to make the comparisons and tests.
          */
 
-        //Method
+        public Game _testGame = new Game();
+        public Die _testDie = new Die();
+
+        //creates a new game and die class
+
+        Debug.Assert(_roll1 >= 1 and _roll1 <= 6);
+        Debug.Assert(_roll2 >= 1 and _roll2 <=6);
+        Debug.Assert(_roll3 >= 1 and _roll3 <=6);
+
+        //tests boundaries of the roll values to ensure all values are between 1 and 6
+
+        Debug.Assert((_roll1 + _roll2 + _roll3) == Game._dieTotal()); 
+
+        //tests that the sum of the die is as expected
```

Comment:
Overall, the code has some flaws but does work as intended, comments are good making it easy to understand and read as a user and it fills all the requriements for the breif

Reply:
Did the project work for you? When I tried running it, it spat out a lot of errors.